src/Controls/Maps/src/Properties/AssemblyInfo.cs

```diff
@@ -1,3 +1,7 @@
+using System.Reflection;
+using Microsoft.Maui.Controls;
 using Microsoft.Maui.Controls.Internals;
 
-[assembly: Preserve]
\ No newline at end of file
+[assembly: Preserve]
+[assembly: XmlnsDefinition("http://schemas.microsoft.com/dotnet/2021/maui", "Microsoft.Maui.Controls.Maps", AssemblyName = "Microsoft.Maui.Controls.Maps")]
```

Comment:
@StephaneDelcroix  is this correct here?

Reply:
I think it enables us to not have to import a separate xmlns:maps thing for it but we can just use it as `<Map />`